### DISCUSSION TITLE
Is there a tool to calc Fibonacci retracement lines?

### DISCUSSION THREAD

**Discussion by Salardx** [DaveSkender/Stock.Indicators#1051] [Q&A]
Hi dave, great work!
We need a tool to be able to calc Fibonacci retracement lines, based on 2 points
Meaning we give it only 2 points (perhaps with couple settings, like Tradingview), it gives us multiple Fibonacci retracement lines
Which one of your tools has such capability?

**Comment by DaveSkender** [comment]
Give [Pivot Points](https://dotnet.stockindicators.dev/indicators/PivotPoints/#content) a try.  There are some other options we can enable quickly too, but you’ll have to provide more info on what you’re seeking.
